lib/commands/description.js

```diff
@@ -42,19 +46,23 @@ if (typeof Program.locale === 'undefined') {
     process.exit(1);
 }
 
-
-// Run program
 const filePath = path.resolve(process.cwd(), Program.args[0]);
-const commander = new ShopwareStoreCommander(Program.username, Program.password);
-fs.exists(filePath)
-    .then((exists) => {
+(async () => {
```

Comment:
Probably better to declare an async function `main()` that contains all the code in this file and then call that just before the EOF.